packages/desktop-client/src/hooks/useAccount.ts

```diff
@@ -0,0 +1,8 @@
+import { useMemo } from 'react';
+
+import { useAccounts } from './useAccounts';
+
+export function useAccount(id: string) {
+  const accounts = useAccounts();
+  return useMemo(() => accounts.find(a => a.id === id), [id]);
```

Comment:
<!-- Warning represent user-facing problems. If possible, it's great to follow this kind of comment with a suggestion.  -->
**🔨 warning:** ‏current implementation won't re-compute if account list changes.
```suggestion
  return useMemo(() => accounts.find(a => a.id === id), [id, accounts]);
```